polaris.shopify.com/src/components/IconsPage/IconsPage.module.scss

```diff
@@ -149,3 +149,20 @@
     color: var(--text-subdued);
   }
 }
+
+.DownloadIconButton {
+  align-items: center;
+  appearance: none;
+  background: var(--primary);
+  color: var(--surface);
+  padding: 0.75rem 0.66rem;
+  border: none;
+  font-size: var(--font-size-100);
+  font-weight: var(--font-weight-500);
+  border-radius: var(--border-radius-400);
+  user-select: none;
+  box-shadow: inset 0 -2px rgba(0, 0, 0, 0.075);
+  display: block;
+  text-align: center;
+  cursor: pointer;
+}
```

Comment:
I wish this was abstracted into another component but not an issue for today.

Reply:
agreed!